sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-storage-blob/sample1/goldenImage/SimpleBlobClientUsage.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: can rename golden image to v2?

Reply:
@samvaity Sure, I'll drop into our chat a poll so we can come up with a name everyone agrees upon.